src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Umbraco.Forms.Integrations.Crm.ActiveCampaign.csproj

```diff
@@ -11,11 +11,12 @@
 		<Title>Umbraco Forms Integrations: CRM - ActiveCampaign</Title>
 		<Description>An extension for Umbraco Forms to add support for submitting contact data to ActiveCampaign</Description>
 		<PackageIconUrl></PackageIconUrl>
-		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Forms.Integrations</PackageProjectUrl>
+		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Forms.Integrations/tree/main-v10/src/Umbraco.Forms.Integrations.Crm.ActiveCampaign</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/umbraco/Umbraco.Forms.Integrations</RepositoryUrl>
 		<Version>1.0.0</Version>
 		<Authors>Umbraco HQ</Authors>
 		<Company>Umbraco</Company>
+		<PackageTags>Umbraco-Marketplace</PackageTags>
```

Comment:
Could add "Umbraco" here too.  Not needed for marketplace but it's common to tag Umbraco packages with this in case people are searching via tags on NuGet.